tests/VolleyManagement.Specs/VolleyManagement.Specs.csproj

```diff
@@ -227,6 +228,10 @@
       <Project>{4D6C885F-9E5D-4F09-B81B-CD14ED6A527F}</Project>
       <Name>VolleyManagement.Services</Name>
     </ProjectReference>
+    <ProjectReference Include="..\VolleyManagement.UnitTests\VolleyManagement.UnitTests.csproj">
```

Comment:
I was talking about all references between test projects.

Reply:
comparers are stored in UT(

Reply:
So what?)

Reply:
Use FluentAssertions